api/helpers/urls.js

```diff
@@ -0,0 +1,14 @@
+function cleanUrlsAndEmails(string) {
+  // this regex finds urls or emails in square brackets, and captures the url
+  // [https://www.example.com|unwanted text] and [example@example|unwanted text/characters]
+  const urlAndEmailRegExPattern =
+    /\[(https?\S+?)\|.*?\]|\[[^\s@]+@[^\s@]+\.[^\s@]+\|.*?\]/g;
+
+  return string.replace(urlAndEmailRegExPattern, (match, url) =>
+    url ? url : ""
```

Comment:
Tiny thing, but you could use a [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) (`??`) here so the line's a bit shorter and neater: `url ?? ''`